docs/django.rst

```diff
@@ -435,7 +474,8 @@ A unique request ID is read from the ``X-Request-ID`` request header, and a UUID
 
 Leveraging the ``RequestIdFilter`` in logging configuration as shown above will add a ``rid`` field into the ``Fields`` entry of all log messages.
```

Comment:
I imagine that this should be ``RequestIdLogFilter`` here too 

Unfortunately `git grep RequestIdFilter` other results :( We'll fix them in a follow-up (unless you want to do them as driveby)